channeldb/invoices.go

```diff
@@ -243,9 +243,24 @@ type InvoiceHTLC struct {
 	Cancelled bool
 }
 
+// InvoiceUpdateDesc describes the changes that should be applied to the
+// invoice.
+type InvoiceUpdateDesc struct {
+	// Htlcs describes the changes that need to be made to the invoice htlcs
+	// in the database. Htlc map entries with their value set should be
+	// added. If the map value is nil, the htlc should be cancelled.
+	Htlcs map[CircuitKey]*InvoiceHTLC
+
+	// State is the new state that this invoice should progress to.
+	State ContractState
+
```

Comment:
What about items like the accepted time? Or is that out of the scope of this PoC, just to demonstrate the possible direction?

Reply:
Which accepted time do you mean? There is no accepted time on the invoice level?